Jenkinsfile

```diff
@@ -135,13 +136,14 @@ node('docker && imgtec') {  // Only run on internal slaves as build takes a lot
                         ])
                     }
                     // Replace (or add if not exists) feed with local clone
-                    sh "grep -q 'src-.* ${feed[0]} .*' feeds.conf.default && \
-                        sed -i 's|^.*\\s\\(${feed[0]}\\)\\s.*\$|src-link \\1 ../feed-${feed[1]}|g' feeds.conf.default || \
-                        echo 'src-link ${feed[0]} ../feed-${feed[1]}' >> feeds.conf.default"
+                    sh "grep -q 'src-.* ${feed[0]} .*' feeds.conf && \
+                        sed -i 's|^.*\\s\\(${feed[0]}\\)\\s.*\$|src-link \\1 ../feed-${feed[1]}|g' feeds.conf || \
+                        echo 'src-link ${feed[0]} ../feed-${feed[1]}' >> feeds.conf"
```

Comment:
@Ham22 with this, the Creator-feed is still missing in feeds.conf.default.  @GonZo expects that when all packages, SDK and ImageBuilder are generated as part of a Ci40 platform release, the Creator packages are also generated and hosted on the downloads.creatordev.io server under packages/creator (or packages/creatordev, please check with him what name he likes).  Without Creator feeds entry in feeds.conf.default (because finally the SDK will also want it to allow building user applications which require Creator packages as dependencies), the Creator IPK packages won't be generated and available on the downloads server as part of our release.